Frontend/src/components/JobListings.js

```diff
@@ -52,15 +52,6 @@ const JobListings = ({ user, setPopupOpen, searchInput = '' }) => {
 
   return (
     <div className='job-listings-container'>
-      <p id='num-listings'>
```

Comment:
Out of curiosity, why was this removed? Wouldn't it be useful to know how many results there were?

Reply:
That is a good question for @Muartin

Reply:
That is a good question